### ISSUE TITLE
Any plans to add Android config?

### ISSUE THREAD

**Issue by CrazyFlasher** [andyli/HaxeCI#9]

**Comment by andyli** [comment]
No. There are too many different approaches to use Haxe for Android. e.g. Use the Java target and build with gradle, or use the cpp target with ndk, or use OpenFL's integration with Android, or use JS target with React Native...
